src/style.css

```diff
@@ -1,3 +1,71 @@
+html {
+  --text-color: #000;
+  --link-color: #00acdc;
+  font-size: 12px;
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+}
+
 body {
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  position: relative;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)